lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -135,12 +136,16 @@ def join_bulk_responses(bulk_responses)
     def bulk_send(body_stream)
       params = http_compression ? {:headers => {"Content-Encoding" => "gzip"}} : {}
       # Discard the URL 
+      @metric.increment(:bulk_requests)
+      @metric.increment(:concurrent_bulk_requests)
```

Comment:
What action would a user take based on this 'concurrent_bulk_requests' value? And could that same action not be informed by the other metric (bulk_requests total count)?